bibliopixel/drivers/network.py

```diff
@@ -56,8 +56,7 @@ def _send_packet(self):
             log.error(error)
             raise IOError(error)
 
-    def set_brightness(self, brightness):
-        super().set_brightness(brightness)
+    def set_hardware_brightness(self, brightness):
```

Comment:
device still implies a physical thing. Why can't this just be set_brightness like every other driver?